docker/autoscaler/README.md

```diff
@@ -0,0 +1,12 @@
+## About
```

Comment:
I would more explicitly mark this as deprecated and say we'll stop pushing it. You can just say `DEPRECATED. This image used to be the base image for the Ray autoscaler, but it has been replaced by rayproject/ray-ml. Please use that instead, this image will be removed in the near future.`

Reply:
Sounds good!